packages/hunspell_fr_fr.rb

```diff
@@ -3,23 +3,23 @@
 class Hunspell_fr_fr < Package
   description 'Hunspell French dictionary'
```

Comment:
Use French language?

Reply:
I don't know french, does anyone else?

Reply:
You don't have to know French with translate.google.com.
```
Dictionnaire Français Hunspell
```

Reply:
As long as it doesn't translate to `Dictionnaire Franco-québécois Hunspell`...

Yeah I don't speak French either...